ui/shared/components/page/Header.jsx

```diff
@@ -24,7 +24,7 @@ const PageHeader = React.memo(({ user }) =>
       user.isStaff ? <Menu.Item key="staff" as={Link} to="/staff" content="Staff Pages" /> : null,
       <Menu.Item key="awesomebar" fitted="vertically"><AwesomeBar newWindow inputwidth="350px" /></Menu.Item>,
       <Menu.Item key="user" position="right">
-        <p>Logged in as &nbsp; <b>{user ? (user.displayName || user.email) : null}</b></p>
+        <p>Logged in as &nbsp; <b>{user && (user.displayName || user.email) + (user.isAnvil ? ' with AnVIL' : '')}</b></p>
```

Comment:
maybe instead of "with AnVIL" do "(AnVIL)"?

Reply:
Hm..., okay